### PR TITLE
Add EIP: Contract Discovery and eTLD+1 Association

### PR DESCRIPTION
**Pull request by TtheBC01**
At Snickerdoodle Labs, we've had good success leveraging whats proposed in this draft EIP in our own protocol. We decided to write it up as a proposal and see if the Ethereum community would also find it useful. 

This proposal describes a simple standard to leverage TXT records to discover smart contracts and verify their association with a known DNS domain. This is enabled by the relatively recent support for DNS over HTTPS (DoH) by most major DNS providers.

### CODE REVIEW

File: EIPS/eip-draft-contract-ownership-via-DoH.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
+---
+eip: XXXX
```
Comment (xinbenlv) [inline EIPS/eip-draft-contract-ownership-via-DoH.md:2]:
```suggestion
eip: 7528
```

**Comment by xinbenlv** [comment]
Hi @TtheBC01 thanks for your draft. ERC Editor here. 
I hereby assigning this ERC as ERC-7528. Please update the filename `EIPS/eip-draft-contract-ownership-via-DoH.md` to be `EIPS/eip-7528.md`

File: EIPS/eip-draft-contract-ownership-via-DoH.md (hunk 1 above)
Comment (TtheBC01) [reply EIPS/eip-draft-contract-ownership-via-DoH.md:2]: thank you, adding now

File: EIPS/eip-7528.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
+---
+eip: 7528
```
Comment (abcoathup) [inline EIPS/eip-7528.md:2]:
```suggestion
eip: 7529
```
Assigning next sequential EIP/ERC number.  7528 already assigned.
EIP/ERC numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.

Comment (TtheBC01) [reply EIPS/eip-7528.md:2]: thank you, on it

**Review by dlgn2 — APPROVED** [review]

**Comment by SamWilsn** [comment]
I am closing this pull request because we are in the process of separating EIPs and ERCs into distinct repositories. Unfortunately, as far as we are aware, GitHub does not provide any tools to ease this migration, so every pull request will need to be re-opened manually.

As this is a PR to create / modify an ERC, I will kindly ask you to redirect this to the new repository at [ethereum/ERCs](https://github.com/ethereum/ERCs). **We have prepared [a guide](https://github.com/ethereum/ERCs/issues/8) to help with the process.**

If there is relevant history here, please link to this PR from the new pull request.

On behalf of the EIP Editors, I apologize for this inconvenience.

**Comment by TtheBC01** [comment]
All good, we'll reopen it in the appropriate place.

**Pull request closed without merging**